sorc/fre-nctools.fd/shared_lib/mosaic.F90

```diff
@@ -1,3 +1,6 @@
+!> @file
+!! author Zhi.Liang@noaa.gov
```

Comment:
This needs to be @author. The "@" is missing.

Reply:
Thanks.  Fixed.